app/controllers/publishers/vacancies/job_applications/self_disclosure_controller.rb

```diff
@@ -0,0 +1,49 @@
+class Publishers::Vacancies::JobApplications::SelfDisclosureController < Publishers::Vacancies::JobApplications::BaseController
+  before_action :not_enabled
+
+  helper_method :job_application
+
+  def show
+    @self_disclosure = SelfDisclosurePresenter.new(job_application)
+    if @self_disclosure.request.manual?
+      @batch = JobApplicationBatch.create!(vacancy_id: job_application.vacancy.id).tap do
```

Comment:
This doesn't appear to be used in any of the views, just it will just leak into the database?